### PR TITLE
Fix table alignment

### PR DESCRIPTION
**Pull request by maya**
This makes sure the first item in the table aligns left like all the other fields. Currently the first item in the table is centered.

Before:

screen shot 2016-04-11 at 3 35 04 pm

After:

screen shot 2016-04-11 at 3 36 16 pm

TODO:

- [x] update other files (`dist` and `docs`) that need this update

### CODE REVIEW

**Comment by maya** [comment]
@ericadeahl @carodew this has design changes, but i'm guessing it was a visual bug before. Can you let me know if this gets a 👍 ?

**Comment by ericadeahl** [comment]
@maya Looks good to me! (And it's helpful to have the screenshot, thanks!)

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+machine:
```
Comment (msecret) [inline circle.yml]: Why are these changes in here?

**Comment by msecret** [comment]
@maya it seems like this has to be rebased.

**Comment by msecret** [comment]
ahh sorry, I closed it by mistake

**Comment by maya** [comment]
@msecret I did rebase it, and also did a pull from `18f-pages-staging`.

File: circle.yml (hunk 1 above)
Comment (maya) [inline circle.yml]: This has the circle CI updates that came after this branch was created.

**Comment by msecret** [comment]
I rebased it

**Pull request merged**